jekyll/_cci2/server-3-install-build-services.adoc

```diff
@@ -520,7 +520,7 @@ This can be found using the following command:
 kubectl get service vm-service --namespace=<YOUR_CIRCLECI_NAMESPACE>
 ```
 ** *GCP project ID (required)* - 
-Name of the GCP project the cluster resides.
+Name of the GCP project the cluster resides. Below this you can uncheck the box if you want to use private VMs, which request private IPs.
```

Comment:
Question....should this be "IP addresses"? Not sure which is why I am asking. There is another instance later in doc as well.

Reply:
you are quite right!